Production/govuk_ios/Helpers/Redactor/Redactables.swift

```diff
@@ -0,0 +1,18 @@
+import Foundation
+
+enum Redactables {
```

Comment:
I think this gets a little confusing...
This is an enum called `Redactables` that returns a `Redactable`. Naming conversions suggest that this enum should be called `Redactable` as well, don't they?

Reply:
I might call the protocol 'Redacting' or 'RedactableProviding', and the enum 'Redactable'.
Or the protocol 'Redactable' and the enum 'Redaction'